components/Seating.js

```diff
@@ -0,0 +1,11 @@
+import styles from "./Seating.module.css";
+const Seating = () => {
+  return (
+    <div className={styles.container}>
+      <p></p>
+      {/* Here be seating module */}
```

Comment:
Yeah, I was looking at it and was like...what's this for...then I sort of lol'd at the ominous formulation.

Reply:
I r pirate